docs/options.md

```diff
@@ -19,6 +19,8 @@ Specify the line length that the printer will wrap on.
 | ------- | --------------------- | ------------------- |
 | `80`    | `--print-width <int>` | `printWidth: <int>` |
 
+(If you don't want line wrapping when formatting Markdown, you can set the [Prose Wrap](#prose-wrap) option to disable it.)
```

Comment:
This `Prose Wrap` casing is weird.. shouldn't it be either `prose-wrap` or `proseWrap` right?